services/movie.py

```diff
@@ -0,0 +1,44 @@
+from db.models import Movie, Actor, Genre
+from typing import List
+
+
+def get_movies(
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]